yarn.lock

```diff
@@ -5,14 +5,12 @@
 "@babel/code-frame@7.0.0", "@babel/code-frame@^7.0.0", "@babel/code-frame@^7.0.0-beta.35":
   version "7.0.0"
   resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.0.0.tgz#06e2ab19bdb535385559aabb5ba59729482800f8"
-  integrity sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==
```

Comment:
you are using old version of yarn can you upgrade it and regenerate yarn lock?

Reply:
I'm using yarn@1.9.4 . I think there is no problem about the version of yarn.
but i cannot generate yarn lock with integrity.

please wait for a little because finding the cause

Reply:
there is problem about the version of yarn ( upgrading from 1.9.4 to 1.10. )
and then it seems to have solved.